aws-attach-roles/README.md

```diff
@@ -0,0 +1,31 @@
+Using aws with IAM roles attached to service accounts:
+
+Enable oidc provider and create service accounts
+```sh
+kubectl create ns kubecost
+eksctl utils associate-iam-oidc-provider \
+    --cluster jesse-eks5 --region us-east-2 \
+    --approve
+eksctl create iamserviceaccount \
```

Comment:
Why don't we just make this for cur/athena access and leave the second one for thanos?

Reply:
i'm good with that